client/cmd/dexc-desktop/pkg/pkg-debian.sh

```diff
@@ -54,18 +39,19 @@ DOT_DESKTOP_BUILDDIR="${DEB_DIR}${DOT_DESKTOP_TARGETDIR}"
 DOT_DESKTOP_FILENAME="dexc.desktop"
 DOT_DESKTOP_BUILDPATH="${DOT_DESKTOP_BUILDDIR}/${DOT_DESKTOP_FILENAME}"
 
-# This will be the icon shown for the program in the taskbar. I know that both
-# PNG and SVG will work. If it's a bitmap, should probably be >= 128 x 128 px.
-ICON_TARGETDIR="/usr/share/pixmaps"
-ICON_BUILDDIR="${DEB_DIR}${ICON_TARGETDIR}"
-DESKTOPICON_BUILDPATH="${ICON_BUILDDIR}/${ICON_FILENAME}"
-
 # Prepare the directory structure.
 rm -fr "${BUILD_DIR}"
 mkdir -p -m 0755 "${CONTROL_DIR}"
-mkdir -p "${SRC_BUILDDIR}" # subdir of BIN_BUILDDIR
 mkdir -p "${DOT_DESKTOP_BUILDDIR}"
-mkdir -p "${ICON_BUILDDIR}"
+
+# check if site bundle is built, if not build it
+if [ ! -d ../../webserver/site/dist ]; then
+  CWD=$(pwd)
+  cd ../../webserver/site
+  npm clean-install
+  npm run build
+  cd $CWD
+fi
```

Comment:
Just because there's a file there doesn't mean its the right one. Any reason not to just build it every time?

Reply:
The idea here was that the site package is added to the repo on the release branch so there's no need to build it at the time of building the release packages but it certainly won't hurt.